README.md

```diff
@@ -441,6 +443,33 @@ temporary file (outside the `source.d` and `features.d` directories), and,
 atomically create/update the original file by doing a filesystem move
 operation.
 
+## Extended resources [experimental]
+
+This feature is experimental and by no means a replacement for the usage of
+device plugins.
+
+Labels which have integer values, can be promoted to Kubernetes extended
+resources by listing them to the master `--resource-labels` command line flag.
+These labels won't then show in the node label section, they will appear only
+as extended resources.
+
+An example use-case for the extended resources could be based on a hook which
+creates a label for the node SGX EPC memory section size. By giving the name of
+that label in the `--resource-labels` flag, that value will then turn into an
+extended resource of the node, allowing PODs to request that resource and the
+Kubernetes scheduler to schedule such PODs to only those nodes which have a
+sufficient capacity of said resource left.
+
+Similar to labels, the default namespace `feature.node.kubernetes.io` is
+automatically prefixed to the extended resource, if the promoted label doesn't
+have a namespace. The default namespace is not stored in node annotations.
```

Comment:
I think the sentence about node annotations could/should be removed as the NFD specific annotations are not mentioned in the document elsewhere.

Just as a reminder to myself we should actually document those. Even if some (like feature labels and extended resources) are sort of "implementation details" of NFD there are the version annotations which might be of more interest to the user. Created #296 to track this